fusedrug/data/tokenizer/modulartokenizer/configs/tokenizer_config.yaml

```diff
@@ -0,0 +1,96 @@
+paths:
```

Comment:
Don't forget to clean this file from the training configuration and full paths in our local system.
Where is the latest and greatest tokenizer config we all expected to use>

Reply:
cleaned local refs